radar/languages_frameworks/angular.yaml

```diff
@@ -0,0 +1,18 @@
+version: 1
+name: Angular
+logo: https://angular.io/assets/images/logos/angular/angular.png
+blip:
+  - version: "2.3"
```

Comment:
Version will be 2.4.
I'd like to find a better solution for this because it is hard to set correctly.